src/Fixer/Phpdoc/PhpdocScalarFixer.php

```diff
@@ -30,6 +30,7 @@
         'boolean' => 'bool',
         'double' => 'float',
         'integer' => 'int',
+        'nullable' => 'null',
```

Comment:
what the hell is `nullable` type? there is no such type nor pseudotype. nullable is attriubte of type since 7.1, like nullable string (`?string)`. nullable without type is meaningless as a type

Reply:
So this change is right? As `PhpdocScalarFixer` is support to fix it :grimacing:

Reply:
How did you get that conclusion?
No, this change is not right. there is no such type or pseudotype as `nullable`.